src/sections/app.style.js

```diff
@@ -3,7 +3,7 @@ import { createGlobalStyle } from "styled-components";
 const GlobalStyle = createGlobalStyle`
 
 ::selection {
-    background: rgba(0, 70, 0, 0.99);
+    background: var(--brand-color-primary);
```

Comment:
```suggestion
    background: ${(props) => props.theme.secondaryColor};
```

Reply:
@Chadha93, this will work.

Reply:
![Screenshot from 2023-06-26 23-29-24](https://github.com/layer5io/layer5/assets/92504666/4a0bd65a-b81b-49bb-bc9d-a841f7d6ffe4)